docs/actions.md

```diff
@@ -116,6 +116,17 @@ Review the following steps and examples to create your first JavaScript action.
   44794bd6aab74415b4e42a308d880e5b         hello
   6bf1f670ee614a7eb5af3c9fde813043         hello
   ```
+7. If you just need access your last activation you can use the flag `--last` or `-l` to retrieve your last activation with get, logs and result. Run the following command to get your last activation result.
```

Comment:
have this step immediately follow after step 5 (both talking about activation results).  suggested language for the text `To access the most recent activation record, activation results or activation logs, use the --last or -l flag.`